tests/drivers/eeprom/api/boards/ucans32k1sic.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Connect I2C0 (connector P4) to the external EEPROM Microchip AT24C01C */
```

Comment:
I understand that this is how you've verified it internally, but this really shouldn't go into the upstream EEPROM test. It adds no real value and would cause the tests to fail on this board unless people made the same modifications to their board. 

I suggest removing the EEPROM test related changes from this PR.

Reply:
Ok removed. Unfortunately this board has a single I2C instance and none of the I2C tests available in upstream can be used, hence I this addition.